tile_pages.py

```diff
@@ -214,15 +216,16 @@ def run(
                 page_names.append(pagekey)
 
                 if abs(pagembox[2] - refmbox[2]) > 1 or abs(pagembox[3] - refmbox[3]) > 1:
-                    print(_('Warning: page {} is a different size from {}, output may be unpredictable').format(p, ref_p))
-
+                    different_size.add(p)
+                
                 refmbox = pagembox
-                ref_p = p
             else:
                 page_names.append(None)
                 pw.append(float(refmbox[2]))
                 ph.append(float(refmbox[3]))
-
+        
```

Comment:
I would add an if statement here to not print anything if different_size is empty.

Reply:
Okay